src/metabase/upload.clj

```diff
@@ -600,7 +600,7 @@
   [fields-by-normed-name header]
```

Comment:
For posterity should we add to this docstring explaining that columns can be in any order?

"the schema of the CSV file does not match the schema of the table" is quite underspecified

Reply:
The `fields-by-normed-name` map is not ordered, but no harm in calling this out.